### ISSUE TITLE
Is there learning-rate decay in each model？

### ISSUE THREAD

**Issue by WelcomeLi** [faneshion/MatchZoo#150]
Can not find interface to set the  learning-rate decay.

**Comment by bwanglzu** [comment]
Currently not supported if you read how the optimizer is initialized in `main`.py

```python
optimizer = global_conf['optimizer']
optimizer=optimizers.get(optimizer)
K.set_value(optimizer.lr, global_conf['learning_rate'])
```

Only `learning rate` was sent an argument to the optimizer. But in `2.0`, we'll have decay in place.

**Comment by bwanglzu** [comment]
Thanks for your question, I'll close it for now. If you have further questions, please feel free to re-open it or create a new issue. Best.
